### PR TITLE
Offline analytics upload endpoint

### PR DESCRIPTION
**Pull request by nous-**
Please consider whether the filename will be correct when uploading to the blob store. Using existing blob container.

### CODE REVIEW

File: api/swagger/swagger.yaml [hunk 1]
```diff
@@ -588,6 +588,30 @@ paths:
...
+      post:
+        operationId: uploadAnalytics
+        description: Uploads offline analytics
+        security:
+          - Bearer: []
+        x-security-scopes:
+          - admin
+          - user
+        consumes:
+          - multipart/form-data
```
Comment (martinbedouret) [inline api/swagger/swagger.yaml]: Is this correct for the data you want to upload ?

File: api/controllers/media.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+const blobService = azure.createBlobService(
+  process.env.AZURE_STORAGE_CONNECTION_STRING
+);
 
 const BLOB_CONTAINER_NAME = process.env.BLOB_CONTAINER_NAME || 'cblob';
 
 module.exports = {
-  uploadMedia: uploadMedia
+  uploadMedia,
+  uploadAnalytics
```
Comment (martinbedouret) [inline api/controllers/media.js]: to me , analytics must be separate from media controller. it needs an own controller

File: api/swagger/swagger.yaml (hunk 1 above)
Comment (nous-) [reply api/swagger/swagger.yaml]: Yes - patterned off media upload

File: api/controllers/media.js (hunk 2 above)
Comment (nous-) [reply api/controllers/media.js]: Seperated

**Pull request merged**